PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskSoftDrop.cxx

```diff
@@ -403,11 +407,19 @@ void AliAnalysisTaskSoftDrop::SoftDropDeepDeclustering(fastjet::PseudoJet jet, c
     Float_t pt1 = jet1.pt();
     Float_t pt2 = jet2.pt();
 
+    Float_t dr = sqrt( jet1.plain_distance(jet2) );
```

Comment:
I think sqrt will needs an explicit namespace (?)

Reply:
I think it's ok as is (was already used in another method). Nevertheless, changed. Thanks.

Reply:
Apologies - I really thought this was going to be an issue on root 6 - I guess I should have tested it! Thanks for bearing with me!